File: get-feature-50.py
#!/usr/bin/env python3
import sqlite3
import json

conn = sqlite3.connect('features.db')
cursor = conn.cursor()

cursor.execute("""
    SELECT id, priority, category, name, description, steps, passes, in_progress, dependencies
    FROM features
    WHERE id = 50
""")

row = cursor.fetchone()
if row:
    feature = {
        'id': row[0],
        'priority': row[1],
        'category': row[2],
        'name': row[3],
        'description': row[4],
        'steps': json.loads(row[5]) if row[5] else [],
        'passes': bool(row[6]),
        'in_progress': bool(row[7]),
        'dependencies': json.loads(row[8]) if row[8] else []
    }
    print(json.dumps(feature, indent=2))
else:
    print("Feature #50 not found")

conn.close()

File: get-feature-50.php
#!/usr/bin/env php
<?php
$db = new SQLite3('features.db');
$result = $db->query("SELECT id, priority, category, name, description, steps, passes, in_progress, dependencies FROM features WHERE id = 50");
$row = $result->fetchArray(SQLITE3_ASSOC);
if ($row) {
    $row['steps'] = json_decode($row['steps'], true);
    $row['dependencies'] = json_decode($row['dependencies'], true);
    echo json_encode($row, JSON_PRETTY_PRINT) . "\n";
} else {
    echo "Feature #50 not found\n";
}
$db->close();
